guides/InstallationAdminGuide.md

```diff
@@ -227,21 +227,19 @@ openEQUELLA provides files for installation that can be used on all platforms. T
 
 This information is required to successfully install openEQUELLA. Incomplete details will cause the installation to fail.
 
-### PLACEHOLDER FOR INSTRUCTIONS ON CREATING/OBTAINING INSTALLATION FILES FOR OPEN openEQUELLA VIA APEREO
-
-1.  ... In Progress...  Need to insert steps required to get install files 
-
 ### Begin openEQUELLA Installation 
 
 This procedure describes installing openEQUELLA using a graphical interface. The examples shown are using Windows although other GUIs such as X-windows will be similar. The wizard pages provide information on the required details. Read each page before entering information.
 
-1. PLACEHOLDER FOR WHERE THE INDIVIDUAL CAN DOWNLOAD THE equella-6.4-GAx Installer file.
-2. Extract the equella-installer-xxxx.zip file to a temporary directory.
+Note: It may be possible to install without requiring a GUI; see https://groups.google.com/a/apereo.org/d/msg/equella-users/v-MMOuoa7mk/UDbmSkKRAwAJ for discussion if this is important to you.
```

Comment:
We can use stronger language, such as:

Note: It is possible to install without requiring a GUI.  While not widespread as a production installation method, it's consistently used in the docker logic. see ....